src/pages/Home/sections/Contact/components/EmojiRating/EmojiRating.module.css

```diff
@@ -24,4 +25,5 @@
 .emoji:hover {
   filter: grayscale(0);
   opacity: 1;
+  transform: rotate(360deg) scale(1.5);
```

Comment:
avoid transforming 360 deg animation instead use grow property with ease in animation that would look more appealing and minimal.